mobilecoind-json/src/data_types.rs

```diff
@@ -841,7 +841,10 @@ impl From<&RingMLSAG> for JsonRingMLSAG {
 pub struct JsonSignatureRctBulletproofs {
     pub ring_signatures: Vec<JsonRingMLSAG>,
     pub pseudo_output_commitments: Vec<String>,
-    pub range_proofs: String,
+    pub range_proof_bytes: String,
+    pub range_proofs: Vec<String>,
```

Comment:
Optional/Equivalent: Add a `_hex` suffix to these new members
```suggestion
    pub range_proof_hex: String,
    pub range_proofs_hex: Vec<String>,
```

Reply:
I think these names impact the Json API, but I'm okay with your suggestions

Reply:
In the rest of this API, we map the protobuf schema onto json names exactly. But actually in this specific spot we had "range_proof" vs "range_proof_bytes"

Reply:
These are strings while the proto fields are bytes, so adding/changing the suffix seems OK

Reply:
With that said, I don't feel strongly about renaming this.